languages/altapay-pl_PL.po

```diff
@@ -0,0 +1,152 @@
+# Copyright (C) 2024 AltaPay
+# This file is distributed under the same license as the AltaPay for WooCommerce - Payments less complicated plugin.
+msgid ""
+msgstr ""
+"Project-Id-Version: AltaPay for WooCommerce - Payments less complicated 3.6.8\n"
+"Report-Msgid-Bugs-To: https://wordpress.org/support/plugin/altapay-for-woocommerce\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: pl_PL\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"POT-Creation-Date: 2024-07-18T10:05:00+00:00\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"X-Generator: WP-CLI 2.10.0\n"
+"X-Domain: altapay\n"
+
+
+#: altapay.php:254
+msgid "AltaPay Payment Actions"
+msgstr "Działania płatnicze AltaPay"
```

Comment:
Not sure of the context here, but this sounds artificial.

If it's title to the section where we list possible actions to be done for given payment then something like that will be better:
```
Operacje na płatności AltaPay
```